README.rst

```diff
@@ -50,6 +50,16 @@ HistomicsTK can be used in two ways:
   and `examples <https://digitalslidearchive.github.io/HistomicsTK/examples.html>`__
   for more information.
 
+  This can be installed via `pip install histomicstk`. 
+
+  If you want to specify precise library versions for reading various
+  microscopy formats, you will need to install various libraries via your
+  package manager or from source (libopenslide-dev, libtiff-dev).
```

Comment:
Does this mean that the wheel ships with Openslide and TIFF library binaries and system ones will be used? Or is it a requirement to install these Linux package manager packages? Clarification in the documentation here would be helpful.

Reply:
The HistomicsTK wheel only contains the built cython code -- everything else is obtained from other packages.  It depends on large_image and requires the openslide and tiff tile sources (as those are the most likely to be useful for microscopy).  Those require *either* that the appropriate libraries are installed on the system, in which case the published python packages to access those libraries work (though with some known bugs) *or* we can use the `--find-links` (`-f`) option in `pip install` to install from my built wheels for python-openslide and libtiff.  In that case, those libraries will use my built versions and NOT any system version (even if so installed).

My wheels for large_image are necessarily not official (as they aren't published by the openslide and pylibtiff projects -- we'd love to get there some day).  To further complicate things, if you try to pip install without the `-f` option and then retry pip install with it, pip will pull from the local cache and not check the find-links address unless you ask to forcibly reinstall AND not use the cache (`--force-reinstall --no-cache-dir`).

The short form is, on linux, installing via `pip install histomicstk -f https://manthey.github.io/large_image_wheels --force-reinstall --no-cache-dir` will always work, but fixes the versions of various libraries in an opinionated manner.  Anything else might not work and requires more explanation.

Recommendations on how to state this clearly and succinctly are welcome.

Reply:
I pushed a change which I hope makes things clearer.